src/utilities/alignment/run_star_and_htseq.py

```diff
@@ -325,13 +403,13 @@ def main(logger):
     else:
         id_attr = "gene_id"
 
-    if args.region != "west" and genome_name not in ("HG38-PLUS", "MM10-PLUS"):
-        raise ValueError(f"you must use --region west for {genome_name}")
-
     genome_dir = os.path.join(root_dir, "genome", "STAR", genome_name)
     ref_genome_star_file = f"STAR/{genome_name}.tgz"
     sjdb_gtf = os.path.join(root_dir, f"{genome_name}.gtf")
```

Comment:
Ah I see I didn't use `pathlib.Path` here (probably written before I learned about it). If you were going to go one way or the other I think pathlib is a lot nicer.